gulpfile.js

```diff
@@ -21,9 +22,10 @@ gulp.task('sass:bootstrap', ['sass:sync'], function () {
     .pipe(stripCssComments({ preserve: false }))
     .pipe(postcss([postcssPrefix({
       prefix: bootstrapNamespace + ' ',
-      exclude: [/^html/, /^body/, /\.ta-hidden-input/, /\.mobile/]
+      exclude: [/^html/, /^body/, /\.ta-hidden-input/, /\.mobile/, outsideNamespaceRegExp]
```

Comment:
Now that we have this special namespace class, i think we can get rid of `.ta-hidden-input` and `.mobile` in this list, if we wrapped it in the namespace

Reply:
@AkA84 `.ta-hidden-input` is inside vendor file, not sure I can do anything with it.
`./scss/bootstrap/vendor/text-angular/_text-angular.scss`